src/pyggdrasil/visualize/_tree.py

```diff
@@ -88,7 +92,7 @@ def plot_tree(
 
     # Calculate an appropriate figure size
     node_width = 0.5  # Width of each node in the figure
-    node_height = 0.75  # Height of each node in the figure
+    node_height = 1.3  # Height of each node in the figure
```

Comment:
adjusting node height - top and bottom nodes were cut off. Apparently, that's nothing Matplotlib can to automatically.